src/CoreBundle/Export/Discovery/EntityDiscovery.php

```
@@ -0,0 +1,277 @@
+                continue;
+            }
+
+            $property = $reflection->getProperty((string) $fieldName);
+            if ($this->hasPropertyIgnoreAttribute($property)) {
+                continue;
+            }
+
+            $included[] = (string) $fieldName;
+        }
+
+        foreach ($metadata->associationMappings as $assocName => $assocMapping) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$assocMapping'. (undefined)

(UnusedLocalVariable)